docs/sdkx-xamarin/performance.mdx

```diff
@@ -55,4 +55,24 @@ When a Xamarin iOS app is integrated with Helpshift plugin, the increase in the
 
 ## 10.2.0 - Android
 
-// TODO
+**Android**
+
+<Admonition type="info" title="Note">
+
+Tested with a demo app on Google Pixel 6a running Android 13.
+
+</Admonition>
+
+#### Disk space
+
+Helpshift Xamarin Android SDK adds around `154 KB` to your final application build file i.e apk file.
+
+### Memory usage {#memory-usage}
+
+| Helpshift API launch                   | Java heap memory (in approx.MB) | Total memory usage(in approx MB) |
+| -------------------------------------- | ------------------------------- | -------------------------------: |
+| showFAQs                               | `1.74`                          |                          `92` |
+| showFAQSection                         | `1.80`                          |                          `81` |
+| showSingleFAQ                          | `1.57`                          |                          `84` |
+| showConversation (conversational mode) | `2.80`                          |                          `78` |
```

Comment:
Don't think (conversational mode) applies to SDK X.

Reply:
fixed